### PR TITLE
fix: MD024 Multiple headings with same content

### PR DESCRIPTION
**Pull request by nschonni**
Took a pass with tightening it up to allow only non-sibling duplicate headings

### CODE REVIEW

File: files/en-us/web/api/element/outerhtml/index.md [hunk 1]
```diff
@@ -48,17 +48,13 @@ descendants with a new DOM tree constructed by parsing the specified
 
 Getting the value of an element's `outerHTML` property:
```
Comment (teoli2003) [inline files/en-us/web/api/element/outerhtml/index.md:49]: I would add an `###` here (and for the other examples) and keep HTML and CSS as `####`.

That way, the different examples are labeled with `###`

This is what we did elsewhere.

File: files/en-us/web/api/gamepad/axes/index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ the axis position from the lowest value (-1.0) to the highest value (1.0).
 
 ## Value
 
-An array.
+An array of {{domxref("double") }} values.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepad/axes/index.md:26]: Number is the primitive type.

```suggestion
An array of numbers.
```

File: files/en-us/web/api/gamepad/id/index.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ device as well as display useful feedback to the user.
 
 ## Value
 
-A string.
+A {{jsxref("String")}}.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepad/id/index.md:34]: It isn't a `String` (the object wrapper), it really is a primitive string.

```suggestion
A string primitive.
```

File: files/en-us/web/api/screen_capture_api/using_screen_capture/index.md [hunk 4]
```diff
@@ -132,10 +132,6 @@ Capturing audio is always optional, and even when web content requests a stream
 
 The {{jsxref("promise")}} returned by {{domxref("MediaDevices.getDisplayMedia", "getDisplayMedia()")}} resolves to a {{domxref("MediaStream")}} that contains at least one video stream that contains the screen or screen area, and which is adjusted or filtered based upon the constraints specified when `getDisplayMedia()` was called.
 
-## Security
-
-As is always the case when sharing content over a network, it's important to consider the privacy and safety implications of screen sharing.
-
```
Comment (teoli2003) [inline files/en-us/web/api/screen_capture_api/using_screen_capture/index.md:138]: Why removing this one?

File: files/en-us/web/api/serviceworkerglobalscope/fetch_event/index.md [hunk 5]
```diff
@@ -30,16 +30,14 @@ addEventListener("fetch", (event) => {});
...
 
-## Event type
-
-A generic {{domxref("Event")}}.
-
-## Syntax
-
 ```js
 serviceWorkerGlobalScope.onfetch = (fetchEvent) => { /* … */ }
 ```
```
Comment (teoli2003) [inline files/en-us/web/api/serviceworkerglobalscope/fetch_event/index.md:36]:
```suggestion
```

File: files/en-us/web/css/css_container_queries/index.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ In addition the draft specification proposes some new properties:
 - `container`:
   - : A shorthand property to set both `container-type` and `container-name`.
 
-### `container-type`
+### `container-type` property
```
Comment (teoli2003) [inline files/en-us/web/css/css_container_queries/index.md:47]: No markdown/HTML in a heading. (Side change)

```suggestion
### The container-type property
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I'm happy to see this one coming to life!

A few changes here or there, but this is close to being merged! 

Thank you!

File: files/en-us/web/api/screen_capture_api/using_screen_capture/index.md (hunk 4 above)
Comment (caugner) [reply files/en-us/web/api/screen_capture_api/using_screen_capture/index.md:138]: Also, `### Potential risks` seemed to have been a subsection of this.

File: files/en-us/web/css/css_container_queries/index.md (hunk 6 above)
Comment (caugner) [reply files/en-us/web/css/css_container_queries/index.md:47]: Wonder if markdownlint has a rule for this?

**Review by caugner — APPROVED** [review]
Config change LGTM

File: files/en-us/web/css/css_container_queries/index.md (hunk 6 above)
Comment (nschonni) [reply files/en-us/web/css/css_container_queries/index.md:47]: @teoli2003 there are others in the file that use code fences as well, which is why this is a duplicate. Is this something new from the contributing guide?

File: files/en-us/web/api/screen_capture_api/using_screen_capture/index.md (hunk 4 above)
Comment (nschonni) [reply files/en-us/web/api/screen_capture_api/using_screen_capture/index.md:138]: There is a detailed section at https://github.com/mdn/content/pull/20118/files#diff-2601f01715acc290cff3cec28a3e85a81dbef5a6e226a2989e14e53726808601R317
It's possible that both are required, but then the heading levels need to be fixed, as there are currently 2 H2 Security headings

Comment (teoli2003) [reply files/en-us/web/api/screen_capture_api/using_screen_capture/index.md:138]: Agreed. This one makes sense, then.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
There is still a comment about `element.outerHTML` to fix.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**